chapters/zh/_toctree.yml

```diff
@@ -0,0 +1,173 @@
+- title: 0. 准备

```

Comment:
We can remove Chapter 0 from the table of contents since it hasn't been translated yet